editor/animation_bezier_editor.cpp

```diff
@@ -1624,15 +1636,23 @@ void AnimationBezierTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 				h += -scaling_selection_offset.y + (h - scaling_selection_pivot.y) * (scaling_selection_scale.y - 1);
 				key[0] = h;
 
+				Vector2 in_handle = Vector2(key[1], key[2]);
+				Vector2 out_handle = Vector2(key[3], key[4]);
+
+				in_handle.x *= scaling_selection_scale.x;
+				in_handle.y *= scaling_selection_scale.y;
+				out_handle.x *= scaling_selection_scale.x;
+				out_handle.y *= scaling_selection_scale.y;
+
```

Comment:
One concern I have is that when scale is inverted, the in and out should also be inverted.